README.md

```diff
@@ -3,26 +3,26 @@
 [![Coverage Status](https://coveralls.io/repos/github/elegant-bro/money/badge.svg?branch=master)](https://coveralls.io/github/elegant-bro/money?branch=master)
 
 # Elegant money implementation
-This library provides classes for money manipulation using pure object-oriented approach.
+This library provides classes for money manipulation using a pure object-oriented approach.
 
 ## Yet another PHP-money, why?
-Why did we make yet another php-library for money? After all there are:
+Why did we make yet another PHP-library for money? After all, there are:
 * https://github.com/moneyphp/money
 * https://github.com/brick/money
 * https://github.com/akaunting/money
 
-But, there are disadvantages for using it in pure object-oriented development.
+But, there are disadvantages to using it in pure object-oriented development.
 
 * There are no interfaces — we can't create an object that will represent money itself.
 * All existed implementations are classes with hundreds loc and dozens of methods like `add`, `sub`,
-  `divide`, `compare` etc., which are essentially a procedural style.
+  `divide`, `compare` etc., which are essentially procedural styles.
```

Comment:
I don't agree with plural

Reply:
vote for `styles` cause words `implementations` and `classes` used before